superset-frontend/src/views/CRUD/utils.test.tsx

```diff
@@ -228,3 +232,165 @@ test('checkUploadExtensions should return valid upload extensions', () => {
     checkUploadExtensions(randomExtensionThree, uploadExtensionTest),
   ).toBeFalsy();
 });
+
+test('getFilterValues', () => {
```

Comment:
While testing I found a bug in my refactored filter logic, so I added tests to make sure this util works as expected in all relevant cases.